### PR TITLE
Cus 33153/add appnexus

### PR DESCRIPTION
**Pull request by ettoreciprianmrf**
Added Appnexus and dimensions to Ligatus

### CODE REVIEW

File: json-lint/schemas/inventory/adservers/appnexus.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+    "$id": "#appnexus",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "title": "Appnexus adserver",
+    "description": "https://github.com/ampproject/amphtml/blob/master/ads/appnexus.md",
+    "additionalProperties": false,
+    "required": [
+        "type"
```
Comment (ricardoribasmrf) [inline json-lint/schemas/inventory/adservers/appnexus.json]: Only type required? Hmmmm

File: json-lint/schemas/inventory/adservers/appnexus.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            }
+        }
+    ],
+    "properties": {
+        "type": {
+            "const": "appnexus"
+        },
+        "tagid": {
+            "type": [
+                "integer",
```
Comment (ricardoribasmrf) [inline json-lint/schemas/inventory/adservers/appnexus.json]: I would put string only.

File: json-lint/schemas/inventory/adservers/appnexus.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                "string"
+            ],
+            "title": "Integer or parameter",
+            "description": "example: 6063968",
+            "pattern": "^(\\$\\{.+\\})$"
+        },
+        "member": {
+            "type": [
+                "integer",
+                "string"
```
Comment (ricardoribasmrf) [inline json-lint/schemas/inventory/adservers/appnexus.json]: Same here

**Comment by ettoreciprianmrf** [comment]
Retest this please

File: json-lint/test/resources/schema/inventory/valid/appnexus.test.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+        "nativeAd": {
+            "adServers": [
+                "appnexus"
+            ]
+        }
+    },
+    "adServers": {
+        "appnexus": {
+            "type": "appnexus",
+            "tagid": 14126014,
```
Comment (ricardoribasmrf) [inline json-lint/test/resources/schema/inventory/valid/appnexus.test.json]: Add tagid as mandatory

File: json-lint/test/resources/schema/inventory/valid/ligatus.test.json [hunk 5]
```diff
@@ -7,7 +7,8 @@
   "adServers": {
     "ligatus": {
       "type": "ligatus",
-      "src": "http://a.ligatus.com/?ids=91606&t=js"
+      "src": "http://a.ligatus.com/?ids=91606&t=js",
```
Comment (ricardoribasmrf) [inline json-lint/test/resources/schema/inventory/valid/ligatus.test.json]: I believe we have more properties from ligatus. Let's add them and make a test for it. Same for appnexus

File: json-lint/schemas/inventory/adservers/appnexus.json (hunk 2 above)
Comment (ettoreciprianmrf) [reply json-lint/schemas/inventory/adservers/appnexus.json]: Can also been a numerical ID in the examples. Added for completion.

File: json-lint/test/resources/schema/inventory/valid/ligatus.test.json (hunk 5 above)
Comment (ettoreciprianmrf) [reply json-lint/test/resources/schema/inventory/valid/ligatus.test.json]: Only SRC: https://github.com/ampproject/amphtml/blob/master/ads/ligatus.md

File: json-lint/schemas/inventory/adservers/appnexus.json (hunk 1 above)
Comment (ettoreciprianmrf) [reply json-lint/schemas/inventory/adservers/appnexus.json]: Added three new fields and one of them is mandatory at least:
https://github.com/ampproject/amphtml/blob/master/ads/appnexus.md

File: json-lint/schemas/inventory/adservers/appnexus.json (hunk 2 above)
Comment (ricardoribasmrf) [reply json-lint/schemas/inventory/adservers/appnexus.json]: Ok. Can live with that 👍 😄

File: json-lint/test/resources/schema/inventory/valid/ligatus.test.json (hunk 5 above)
Comment (ricardoribasmrf) [reply json-lint/test/resources/schema/inventory/valid/ligatus.test.json]: 👍

**Pull request merged**